src/rnn_no_discrim.json

```diff
@@ -1,26 +1,29 @@
 {
     "seed": 0,
-    "epochs": 100,
+    "epochs": 300,
     "lr": 1e-3,
     "weight_decay": 1e-6,
     "lr_milestones": [
-        70,
-        90
+        100,
+        150
     ],
     "lr_gamma": 1e-2,
     "teacher_init_val": 1,
     "teacher_gamma": 0.98,
-    "teacher_decay_start": 10,
-    "teacher_decay_end": 70,
+    "teacher_decay_start": 50,
+    "teacher_decay_end": 200,
     "grad_clip": 1.0,
-    "batch_size": 16,
+    "train_batch_size": 4,
```

Comment:
Might be able to try a larger batch size no with the:
`with torch.no_grad():`
in inference!  I can reliably do 16